indexclient/client.py

```diff
@@ -116,6 +115,24 @@ def bulk_request(self, dids):
             for doc in response.json()
         ]
 
+    def bulk_get_latest(self, dids, skip_null=False):
```

Comment:
Can you add docs with type specified, and a check to ensure `dids` is a list

Reply:
I can add document. But for the check if dids is a list is already in indexd, I want to make this as thin as possible. If some 
not json serializable stuff is passed in, then it will just raised here.